### PR TITLE
Add further Terminal translation

### PR DESCRIPTION
**Pull request by Quarri6343**
**What:**
there are a few apps have no translation key
this PR fixes them

**Outcome:**
people will not get lost in the English app

**Additional info:**
I didn't translate ru_ru.lang

**Possible compatibility issue:**
none

### CODE REVIEW

File: src/main/java/gregtech/common/terminal/app/game/maze/MazeApp.java [hunk 1]
```diff
@@ -60,13 +61,13 @@ public AbstractApplication initApp() {
...
+            this.addWidget(new ClickButtonWidget(323 / 2 - 10, 222 / 2 - 10, 50, 20, "terminal.maze.continue", (clickData) -> this.setGameState(1)).setShouldClientCallback(true), 2);
+            this.addWidget(new LabelWidget(333 / 2, 222 / 2 - 50, "terminal.maze.pause", 0xFFFFFFFF).setXCentered(true), 2);
             // GameState 3: Death
-            this.addWidget(new SimpleTextWidget(333 / 2, 232 / 2 - 40, "", 0xFFFFFFFF, () -> "Oh no! You were eaten by the Minotaur!", true), 3);
-            this.addWidget(new SimpleTextWidget(333 / 2, 232 / 2 - 28, "", 0xFFFFFFFF, () -> "You got through " + this.getMazesSolved() + " mazes before losing.", true), 3);
-            this.addWidget(new SimpleTextWidget(333 / 2, 232 / 2 - 16, "", 0xFFFFFFFF, () -> "Try again?", true), 3);
-            this.addWidget(new ClickButtonWidget(323 / 2 - 10, 222 / 2 + 10, 40, 20, "Retry", (clickData -> {
+            this.addWidget(new SimpleTextWidget(333 / 2, 232 / 2 - 40, "", 0xFFFFFFFF, () -> "terminal.maze.death.1", true), 3);
+            this.addWidget(new SimpleTextWidget(333 / 2, 232 / 2 - 28, "", 0xFFFFFFFF, () -> I18n.format("terminal.maze.death.2",this.getMazesSolved()) , true), 3);
```
Comment (brachy84) [inline src/main/java/gregtech/common/terminal/app/game/maze/MazeApp.java:68]: Did you test this on server? It probably will crash

**Review by brachy84 — CHANGES_REQUESTED** [review]

File: src/main/java/gregtech/common/terminal/app/game/maze/MazeApp.java (hunk 1 above)
Comment (Quarri6343) [reply src/main/java/gregtech/common/terminal/app/game/maze/MazeApp.java:68]: I own no server and I totally forgot I18i is client only
is there any better solution?

Comment (brachy84) [reply src/main/java/gregtech/common/terminal/app/game/maze/MazeApp.java:68]: just remove it
the widget should auto translate it

Comment (brachy84) [reply src/main/java/gregtech/common/terminal/app/game/maze/MazeApp.java:68]: wait, i see
In the `""` put the lang key and in the supplier put the argument

Comment (Quarri6343) [reply src/main/java/gregtech/common/terminal/app/game/maze/MazeApp.java:68]: got it, thanks!
I also found it states itself a client only app tho
```
    @Override
    public boolean isClientSideApp() {
        return true;
    }
```

**Review by brachy84 — APPROVED** [review]
Looks good. I did not test ingame

**Comment by ALongStringOfNumbers** [comment]
This needs to be rebased after the `ru_ru` lang update was merged

**Comment by Quarri6343** [comment]
the rebase has been completed!

**Pull request merged**